### PR TITLE
Add press mentions of TWC

### PR DESCRIPTION
**Pull request by shushugah**
Add press mentions of TWC and update documentation, can be updated later with image of newspapers, and their twitter handles later

### CODE REVIEW

File: press_mentions.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+namespace: press_mentions
+permalink: /press
+permalink_de: /pressespiegel
+---
+# {% t press.mentions %}
+
+{% t press.inquiry %}: press@techworkersberlin.com
+
+<ul class=" article-list list">
+  {% assign sorted_posts = site.data.press  %}
```
Comment (shushugah) [inline press_mentions.md]: this needs to be sorted

File: _i18n/en.yml [hunk 2]
```diff
@@ -93,3 +93,7 @@ en:
       - October
       - November
       - December
+
+press:
+  mentions: Press mentions
+  inquiry: For Press inquiries contacts us at
```
Comment (mleimeister) [inline _i18n/en.yml]: contacts -> contact?

File: _i18n/de.yml [hunk 3]
```diff
@@ -91,3 +91,7 @@ de:
       - Oktober
       - November
       - Dezember
+
+press:
+  mentions: Pressespiegel
+  inquiry: Bitte kontaktieren Sie uns für Pressenafragen hier
```
Comment (mleimeister) [inline _i18n/de.yml]: Pressenafragen->Presseanfragen

File: _data/press.yml [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+- media: Jacobin Magazin
+  lang: DE
+  url: https://jacobin.de/artikel/n26-betriebsrat-worker26-start-up/
+  title: "Wie N26 versucht, einen Betriebsrat zu verhindern"
+  date: 2020-08-28
+  twitter: jacobinmag_de
+
+
+- media: Tagespegial
```
Comment (mleimeister) [inline _data/press.yml]: Tagesspiegel

**Review by mleimeister — APPROVED** [review]
Looks great! Just commented on some typos. 

If we could sort the articles by date automatically would be nice, but could also add them in order to the top of the file.

In order to get to the page from the starting page, would a link to the press mentions (and also news/articles) in the header (next to events and learn) be an option?

**Pull request merged**